builds/e2e/connectivity.yaml

```diff
@@ -144,7 +144,7 @@ jobs:
           buildType: specific
           project: $(vsts.project)
           pipeline: $(edgelet.package.build)
```

Comment:
`project` and `pipeline` should also come from resources, rather than defining extra variables in the pipeline. See how we do it in the [end-to-end pipeline](https://github.com/Azure/iotedge/blob/release/1.2/builds/e2e/templates/e2e-setup.yaml#L42-L43)

Reply:
Good call. I've made the change and kicked off this pipeline to confirm it works: https://msazure.visualstudio.com/One/_build/results?buildId=46541636&view=results